.travis.yml

```diff
@@ -22,20 +22,7 @@ cache:
   directories:
     - node_modules
 
-before_install:
```

Comment:
@nathanstitt Why did you remove kcov? Was it not building?

Reply:
I think that could be re-added.  I'd removed it because I thought it was the cause of the failures, but I think the actual cause was the fix in https://github.com/openstax/tutor-js/pull/1991/commits/05d1fc0de9b1e8375a0c76fa3ae4e7de2c4b17b4

However is it the case that `konv` is for coverage of the shell scripts?  In that case I don't think it adds any value since we don't have any specs for the scripts at all and it takes quite awhile to compile on every Travis run.